src/main/java/org/spongepowered/common/event/tracking/phase/GeneralPhase.java

```diff
@@ -147,7 +149,13 @@ public GeneralPhase addChild(TrackingPhase child) {
     public void unwind(CauseTracker causeTracker, IPhaseState state, PhaseContext phaseContext) {
         if (state == State.COMMAND) {
             final ICommandSender sender = phaseContext.getSource(ICommandSender.class)
```

Comment:
Given that the Sponge command manager will always insert the `CommandSource` as source and the code does nothing except some `instanceof` checks, it should be safe to just change this to `CommandSource.class` directly here. (Because the source will **always** be a `CommandSource` but not necessarily always a `ICommandSender`)